org.srb2.SRB2Kart.metainfo.xml

```diff
@@ -14,10 +14,6 @@
       and a Battle mode where players fight each other using the items
       available in the game.
     </p>
-    <p>
```

Comment:
This notice was added as a result of the peer review by other Flathub maintainers. The only modification that I agree with is to replace `commercial` with `proprietary`.

Reply:
We can remove it now that we provide the assets through extra-data :+1:

Reply:
yes sorry that was my intention with this MR (after extra-data)